README.md

```diff
@@ -68,6 +68,8 @@ require('nvim-lsp-setup').setup({
     -- gi = 'lua require"telescope.builtin".lsp_implementations()',
     -- gr = 'lua require"telescope.builtin".lsp_references()',
     mappings = {},
+    -- configurable format_on_save timeout
+    format_on_save_timeout = 1000,
```

Comment:
IMHO, we can still use `on_attach` instead of a new option.

WDYT?

Reply:
Yes, that would be preferable, but it seems like `format_on_save` is called outside the `on_attach` now. https://github.com/junnplus/nvim-lsp-setup/blob/main/lua/nvim-lsp-setup/init.lua#L11

Seems like this commit changed it: https://github.com/junnplus/nvim-lsp-setup/commit/ceefba1938601e9609c39aba12b269f99b6a5dcd

Can you revert that so it's called inside? Thanks!

Reply:
I added a PR for that: https://github.com/junnplus/nvim-lsp-setup/pull/26

Reply:
I don't understand why to revert it, the default is to support formatting on save.

If you want to change the timeout of format, just set `on_attach` to overwrite the default.

Reply:
Thanks